mojaloop/Chart.yaml

```diff
@@ -14,106 +14,114 @@ maintainers:
   - name: Tom Daly
     email: tomd@crosslaketech.com
 dependencies:
-  - name: central
-    version: 14.1.0
-    repository: "file://../central"
-    condition: central.enabled
-  - name: ml-api-adapter
-    version: 12.1.0
-    repository: "file://../ml-api-adapter"
-    condition: ml-api-adapter.enabled
   - name: account-lookup-service
-    version: 13.1.0
+    version: 14.0.0
     repository: "file://../account-lookup-service"
     condition: account-lookup-service.enabled
   - name: quoting-service
-    version: 13.1.0
+    version: 14.0.0
     repository: "file://../quoting-service"
     condition: quoting-service.enabled
+  - name: ml-api-adapter
+    version: 13.0.0
+    repository: "file://../ml-api-adapter"
+    condition: ml-api-adapter.enabled
+  - name: centralledger
+    version: 14.0.0
+    repository: "file://../centralledger"
+    condition: centralledger.enabled
+  - name: centralsettlement
+    version: 14.1.0
+    repository: "file://../centralsettlement"
+    condition: centralsettlement.enabled
+  - name: centraleventprocessor
+    version: 13.0.0
+    repository: "file://../centraleventprocessor"
+    condition: centraleventprocessor.enabled
   - name: simulator
-    version: 12.1.0
+    version: 13.0.0
     repository: "file://../simulator"
     condition: simulator.enabled
   - name: mojaloop-simulator
     version: 14.1.0
     repository: "file://../mojaloop-simulator"
     condition: mojaloop-simulator.enabled
   - name: mojaloop-bulk
-    version: 14.1.0
+    version: 15.0.0
     repository: "file://../mojaloop-bulk"
     condition: mojaloop-bulk.enabled
   - name: transaction-requests-service
-    version: 12.1.0
+    version: 13.0.0
     repository: "file://../transaction-requests-service"
     condition: transaction-requests-service.enabled
   - name: emailnotifier
```

Comment:
are we removing this and CEP here or elsewhere?

Reply:
I added a task to the main v15 release story to address both the CEP and the Email-Notifier.